server/Models/DTOs/Request/CreateBookingRequest.cs

```diff
@@ -14,6 +13,7 @@ public class CreateBookingRequest
         public List<int>? SeatList { get; set; }
 
         public bool? IsEvent { get; set; } = false;
+        public string? reCAPTCHAToken { get; set; }
```

Comment:
hva svare backend dersom vi ikke sende inn noe reCAPTCHAToken ?

Reply:
`                    throw new Exception("reCAPTCHA token is missing or empty.");
`